### ISSUE TITLE
Shell command failure on first project

### ISSUE THREAD

**Issue by jwoertink** [gleam-lang/gleam#2404]
Hey, just installed Gleam, and when I try to do anything, I get this error...

```
❯ gleam new sandbox
Your Gleam project sandbox has been successfully created.
The project can be compiled and tested by running these commands:

	cd sandbox
	gleam test


~/Development/gleam 
❯ cd sandbox/

sandbox on  main [?] 
❯ gleam test
  Resolving versions
Downloading packages
 Downloaded 2 packages in 0.03s
  Compiling gleam_stdlib
  Compiling gleeunit
/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:6:14: can't find include lib "eunit/include/eunit.hrl"
.....

❯ gleam run
  Compiling gleeunit
/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:6:14: can't find include lib "eunit/include/eunit.hrl"
%    6| -include_lib("eunit/include/eunit.hrl").
%     |              ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:14:6: undefined macro 'assertEqual/2'
%   14|     ?assertEqual(Expected, Actual),
%     |      ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:17:6: undefined macro 'assertNotEqual/2'
%   17|     ?assertNotEqual(Expected, Actual),
%     |      ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:20:6: undefined macro 'assertMatch/2'
%   20|     ?assertMatch({ok, _}, A),
%     |      ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:23:6: undefined macro 'assertMatch/2'
%   23|     ?assertMatch({error, _}, A),
%     |      ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:3:2: function should_be_error/1 undefined
%    3| -export([find_files/2, should_equal/2, should_not_equal/2, should_be_ok/1,
%     |  ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:3:2: function should_be_ok/1 undefined
%    3| -export([find_files/2, should_equal/2, should_not_equal/2, should_be_ok/1,
%     |  ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:3:2: function should_equal/2 undefined
%    3| -export([find_files/2, should_equal/2, should_not_equal/2, should_be_ok/1,
%     |  ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_ffi.erl:3:2: function should_not_equal/2 undefined
%    3| -export([find_files/2, should_equal/2, should_not_equal/2, should_be_ok/1,
%     |  ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_progress.erl:10:14: can't find include lib "eunit/include/eunit.hrl"
%   10| -include_lib("eunit/include/eunit.hrl").
%     |              ^

/home/jeremy/Development/gleam/sandbox/build/dev/erlang/gleeunit/_gleam_artefacts/gleeunit_progress.erl:8:2: Warning: behaviour eunit_listener undefined
%    8| -behaviour(eunit_listener).
%     |  ^

error: Shell command failure

There was a problem when running the shell command `escript`.

```

I installed Gleam through asdf

```
❯ gleam --version
gleam 0.32.3
❯ erl -version
Erlang (SMP,ASYNC_THREADS) (BEAM) emulator version 12.2.1
❯ neofetch
             /////////////                jeremy@pop-os 
         /////////////////////            ------------- 
      ///////*767////////////////         OS: Pop!_OS 22.04 LTS x86_64 
    //////7676767676*//////////////       Host: Thelio Mira thelio-mira-r1 
   /////76767//7676767//////////////      Kernel: 6.5.6-76060506-generic 
  /////767676///*76767///////////////     Uptime: 1 day, 28 mins 
 ///////767676///76767.///7676*///////    Packages: 3064 (dpkg), 58 (flatpak), 19 (snap) 
/////////767676//76767///767676////////   Shell: bash 5.1.16 
//////////76767676767////76767/////////   Resolution: 1920x1080, 3840x2160 
///////////76767676//////7676//////////   DE: GNOME 42.5 
////////////,7676,///////767///////////   WM: Mutter 
/////////////*7676///////76////////////   WM Theme: Pop 
///////////////7676////////////////////   Theme: Desert-GTK [GTK2/3] 
 ///////////////7676///767////////////    Icons: pop-os-branding [GTK2/3] 
  //////////////////////'////////////     Terminal: gnome-terminal 
   //////.7676767676767676767,//////      CPU: AMD Ryzen 5 5600X (12) @ 3.700GHz 
    /////767676767676767676767/////       GPU: NVIDIA GeForce RTX 3070 Ti 
      ///////////////////////////         Memory: 5502MiB / 64202MiB 
         /////////////////////
             /////////////                                        
                                                                  
```

**Comment by lpil** [comment]
Hello! Your Erlang install is missing the eunit component so the Erlang code is failing to compile. This normally happens when installed via apt as for some reason the package maintainers decided to remove much of the official Erlang distribution from their package and place them in secondary packages.

**Comment by jwoertink** [comment]
Oh, that's interesting. I don't actually have erlang installed through apt though :thinking:  I went to the Erlang site and it recommends apt https://www.erlang.org/downloads. I uninstalled erlang, and tried using the instructions on the gleam website under "Installing Erlang" 
![image](https://github.com/gleam-lang/gleam/assets/2391/8de3606a-92e8-4ffc-8c2b-4c4f1f700ca0)
 
but that install fails for me. 

I'll keep messing around and make sure I have eunit installed. Thanks for the help!
